src/xdb/process.ts

```diff
@@ -0,0 +1,25 @@
+import {StateSchema} from "./state-schema";
+import {PersistenceSchema} from "./persistence-schema";
+import {ProcessIdReusePolicy} from "xdb-ts-api";
+
+export interface Process {
+    // processType is a required field to identify the process type.
+    // It must be unique in all the processes registered in the same registry.
+    // It's recommended to just use the Class name of the process as the processType.
+    processType: string
```

Comment:
@lefttree in Java/Golang/Python SDKs, processType/stateId are optional because they can be set automatically using reflection (like this https://github.com/xdblab/xdb-java-sdk/blob/main/src/main/java/io/xdb/core/utils/ProcessUtil.java#L20) . But looks like there is no way to get a class name in typescript/javascript.